com.unity.netcode.gameobjects/Runtime/NetworkVariable/ClientNetworkVariable.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+
+namespace Unity.Netcode
+{
+    /// <summary>
+    /// A ClientNetworkVariable is special in that:
+    ///  - only the owner of the variable can write to it
+    ///  - not even the server can write to it
+    ///  - it is not snapshotted
```

Comment:
I thought the direction we were taking was for everything to be snapshotted. AFAIK the client can send snapshots to the server as well.